omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -42,7 +42,7 @@
       add_header X-Content-Type-Options nosniff;
       add_header X-XSS-Protection 1;
       <% if @server_proto == 'https' %>
-      add_header Strict-Transport-Security max-age=31536000;
+      add_header Strict-Transport-Security max-age=<%= node['private_chef']['nginx']['sts_max_age'] %>;
```

Comment:
Included the 'includeSubDomains'. It seems that 'always' parameter is not supported in the current NGINX version.

The headers are add to the manage urls as well. The check was add to avoid duplicate headers according tot he commit https://github.com/chef/chef-server/commit/0f6a11893053bf9310d44d88861888697eaf4edd